running-locally.adoc

```diff
@@ -1,64 +1,46 @@
 [[running-the-demo-raincatcher-solution-locally]]
-Running The Demo Raincatcher Solution Locally
----------------------------------------------
-
-[[overview]]
-Overview
-~~~~~~~~
-
-This guide presents the steps necessary to:
-
-* Run the Raincatcher demo solution locally.
-* Edit Raincatcher modules and have the results reflected in the demo
-solution.
+= Running The Demo Raincatcher Solution Locally
 
 [[requirements]]
-Requirements
-~~~~~~~~~~~~
+== Requirements
 
 [[rhmap]]
-RHMAP
-~~~~~
+=== RHMAP
 
-The demo solution currently requires an instance of RHMAP and will not work properly without it. It can be a local or remote instance of RHMAP, but must be configured as outlined in the Raincatcher link:getting-started.adoc[Getting Started Guide]. This allows the local *raincatcher-demo-cloud* app to access
```

Comment:
either 
a) readers  read our requirements and make sure they have an instance of rhmap
b) don't read our requirements, and cannot get things working
Normally what happens is b) and then a) followed by a request that the requirement be put in **BOLD**
I think this is clear, the only way a user could be surprised is because of the title 'running raincatcher locally', there is a case to be made that this title is misleading?